lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -6,6 +6,15 @@ module Contrib
     # Rack module includes middlewares that are required to trace any framework
     # and application built on top of Rack.
     module Rack
+      # RACK headers to test when doing distributed tracing.
+      # They are slightly different from real headers as Rack uppercases everything
+
+      # Header used to transmit the trace ID.
+      HTTP_HEADER_TRACE_ID = 'HTTP_X_DATADOG_TRACE_ID'.freeze
+
+      # Header used to transmit the parent ID.
+      HTTP_HEADER_PARENT_ID = 'HTTP_X_DATADOG_PARENT_ID'.freeze
```

Comment:
I am already sending these under the old name, it won't require too much change on my side but it would be nice if they were configurable.